gradle.properties

```diff
@@ -1,15 +1,16 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=1.0.6-SNAPSHOT
-ballerinaLangVersion=2.0.0-Preview7
+ballerinaLangVersion=2.0.0-Preview7-SNAPSHOT
```

Comment:
Shall we update to the next SNAPSHOT versions? lang and the stdlibs as well?

Reply:
Fixed